contracts/handler/CompatibilityFallbackHandler.sol

```diff
@@ -124,16 +125,15 @@ contract CompatibilityFallbackHandler is TokenCallbackHandler, ISignatureValidat
                     // address() has been changed to caller() to use the implementation of the Safe
                     caller(),
                     0,
-                    internalCalldata,
+                    ptr,
                     calldatasize(),
                     /**
                      * The `simulateAndRevert` call always reverts, and
                      * instead encodes whether or not it was successful in the return
                      * data. The first 32-byte word of the return data contains the
-                     * `success` value, so write it to memory address 0x00 (which is
-                     * reserved Solidity scratch space and OK to use).
+                     * `success` value, so write it to `ptr`.
                      */
-                    0x00,
+                    add(ptr, calldatasize()),
```

Comment:
THis should be 0 - we write 32 bytes to the scratch space for just the 32-byte "boolean" value (representing if the call reverted or not) which is memory safe.

Reply:
Done 👍🏾